gatsby/content/projects/clients/chooj.mdx

```diff
@@ -0,0 +1,31 @@
+---
+layout: projectimage
+title: chooj
+categories:
+# valid categories are client, server, sdk (client sdk), bot, bridge, iot and other
+ - client
+description: Matrix client for KaiOS
+slug: chooj
+author: Farooq Karimi Zadeh
+home: https://github.com/farooqkz/chooj
+language: Javascript
+license: GPL-3.0-or-later
+# pick your license from https://spdx.org/licenses/
+repo: https://github.com/farooqkz/chooj # your source code repository
+room: "#chooj:mozilla.org" # nb the quotes are needed as rooms start with #
+thumbnail: /docs/projects/images/chooj-screenie.png
+screenshot: /docs/projects/images/chooj-screenie.png
+maturity: Not actively maintained # Options are: Released, Stable, Late Beta, Beta, Early Beta, Late Alpha, Alpha, Early Alpha, or Not actively maintained
+SDK: matrix-js-sdk
+platforms:
+    - KaiOS
+features:
+    Send read markers: yes
+    Display read markers: yes
+    Typing Notification: yes
+    Replies: no
+    voip: yes
+    SSO Support: no
```

Comment:
It's not strictly mandatory, but would you mind adding the full set of features :) ?

Reply:
sure where is the reference? any besides yes and no do you have a `paritally yes` option? For example for VoIP the software supports outgoing voice calls but no incoming calls.

Reply:
You can take a look at what [Element does](https://github.com/matrix-org/matrix.org/blob/master/gatsby/content/projects/clients/element.mdx) for reference

Reply:
As for the partially implemented, those would count as `no`
`yes` is for a feature fully supported

Reply:
Should I include the features my client does not support and set those to `no`? or not including them means already no?

Reply:
Please include them. I'm trying to tidy up data on matrix.org, this will make things easier

Reply:
done. please let me know if any other change is required or recommended.